qcodes/station.py

```diff
@@ -313,6 +334,41 @@ def get_config_file_path(
         with open(path) as f:
             self.load_config(f)
 
+    def load_config_files(self,
+                          *filenames: str
+                          ) -> None:
+        """
+        Loads configuration from multiple YAML files after merging them
+        into one. If `filenames` are not specified the default file name from
+        the qcodes configuration will be used.
+
+        Loading of configuration will update the snapshot of the station and
+        make the instruments described in the config files available for
+        instantiation with the :meth:`load_instrument` method.
+
+        Additionally the shortcut methods ``load_<instrument_name>`` will be
+        updated.
+        """
+        if len(filenames) == 0:
+            self.load_config_file()
+        else:
+            paths = list()
+            for filename in filenames:
+                assert isinstance(filename, str)
+                path = self._get_config_file_path(filename)
+
+                if path is None and filename is not None:
+                    raise FileNotFoundError(path)
+
+                paths.append(path)
+
+            if None in paths:
+                raise RuntimeWarning("Couldn't find path to one or more of the "
```

Comment:
isn't this case taken care of by ``raise FileNotFoundError(path)`` line earlier?

Reply:
yes, you are right. removing it.